package-lock.json

```diff
@@ -1,3989 +1,8 @@
 {
   "name": "opensearch-dashboards-functional-test",
   "version": "3.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
-    "": {
-      "name": "opensearch-dashboards-functional-test",
-      "version": "3.0.0",
-      "hasInstallScript": true,
-      "license": "ISC",
-      "dependencies": {
-        "@cypress/skip-test": "^2.6.1",
-        "@opensearch-dashboards-test/opensearch-dashboards-test-library": "https://github.com/opensearch-project/opensearch-dashboards-test-library/archive/refs/tags/1.0.6.tar.gz",
-        "brace": "^0.11.1",
```

Comment:
lots of removal in this file, is it by intention?

Reply:
Hi, thanks for reminding, I added a new dependency to import files from fixture, this will remove unused dependency when running import test, already recovered from main